liblepton/src/g_register.c

```diff
@@ -43,26 +43,26 @@ struct gsubr_t {
 
 /*! \brief */
 static struct gsubr_t libgeda_funcs[] = {
-  { "eval-protected",            1, 1, 0, (SCM (*) ()) g_scm_eval_protected },
-  { "eval-string-protected",     1, 0, 0, (SCM (*) ()) g_scm_eval_string_protected },
+  { (char*) "eval-protected",            1, 1, 0, (SCM (*) ()) g_scm_eval_protected },
```

Comment:
Probably you can fix all of these by changing `gsubr_t::name` from `char* name` to `const char* name`.